### ISSUE TITLE
Notice in debug mode

### ISSUE THREAD

**Issue by jbonlinea** [agileware/wp-civicrm-ux#9]
Hi, 

it's probable very minor but it happens that I get this notice in debug mode, on evey front page, even if no civicrm-ux shortcode are found there.

regards
```
Notice: Undefined index: civicrm_contribution_ux in /wp-content/plugins/wp-civicrm-ux/includes/class-civicrm-ux-option-store.php on line 41

Notice: Trying to access array offset on value of type null in /wp-content/plugins/wp-civicrm-ux/includes/class-civicrm-ux-option-store.php on line 41

Notice: Trying to access array offset on value of type null in /wp-content/plugins/wp-civicrm-ux/public/class-civicrm-ux-public.php on line 105

Notice: Trying to access array offset on value of type null in /wp-content/plugins/wp-civicrm-ux/public/class-civicrm-ux-public.php on line 106
```

**Comment by agileware-justin** [comment]
Thanks for reporting, I don't think we've seen this problem yet on sites where the plugin is being used.

Have you tried updating to the latest release?

**Comment by jbonlinea** [comment]
hi, 
well no because #12 
so I'm still using wp-civicrm-ux 1.1.2

**Comment by agileware-justin** [comment]
That should be fixed in the latest commit and in 1.8.2

**Comment by jbonlinea** [comment]
:+1: 
great thank's
